lib/ansible/modules/storage/purestorage/purefa_pg.py

```diff
@@ -119,10 +131,12 @@ def get_pgroup(module, array):
 
 def make_pgroup(module, array):
 
-    changed = True
+    changed = False
 
     if not module.check_mode:
+        changed = True
```

Comment:
`changed` should be the same value, no matter if in check mode or not. It says "this ressource will change in checkmode" and "this ressource changed" in live mode.